packages/jest-diff/yarn.lock

```diff
@@ -1,45 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-# yarn lockfile v1
-
-
-ansi-regex@^2.0.0:
-  version "2.1.1"
-  resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-2.1.1.tgz#c3b33ab5ee360d86e0e628f0468ae7ef27d654df"
-
-ansi-styles@^2.2.1:
```

Comment:
Just read you description again @bestander and some of the RFCs. I guess removing the lockfiles is what you means by

> Unlike Lerna Yarn hoists workspaces to the root node_modules.

: )